detection-rules/link_suspicious_message_unscannable_cloudflare.yml

```diff
@@ -231,48 +234,88 @@ source: |
   )
   
   // link can't be scanned due to Cloudflare captcha
-  and any(body.links,
-          (
-            strings.icontains(ml.link_analysis(., mode="aggressive").final_dom.display_text,
-                              "cloudflare"
-            )
-            // includes the turnstile CAPTCHA
-            or (
-              strings.icontains(ml.link_analysis(., mode="aggressive").final_dom.raw,
-                                'https://challenges.cloudflare.com/turnstile/'
-              )
-              // has a short body length indicating the page is gated behind the turnstile instead
-              // of just including the turnstile
-              and length((
-                           ml.link_analysis(., mode="aggressive").final_dom.display_text
-                         )
-              ) < 200
+  and (
+    any(body.links,
+        (
+          strings.icontains(ml.link_analysis(., mode="aggressive").final_dom.display_text,
+                            "cloudflare"
+          )
+          // includes the turnstile CAPTCHA
+          or (
+            strings.icontains(ml.link_analysis(., mode="aggressive").final_dom.raw,
```

Comment:
you might also be able to use the `unique_urls_accessed` and look for URls going to cloudflare with captcha resources?

Reply:
Yeah, this is likely the same effect, but good call